SpringVerify US API doc.md

```diff
@@ -0,0 +1,43 @@
+# Welcome!
+
+Hello and welcome to SpringVerify.
+
+This is the API documentation section of SpringVerify for customers and users located in the USA. It is divided into three sections for ease of navigation:
+1. Introductions - _where entities, environments and authentication protocals are detailed._
+2. Company API flow - _where functions available to users logged in as company admins are detailed._
+3. User API flow - _where functions available to users logged in as employees are detailed._
+
+## Introduction
+
+SpringVerify currently has two environments -- development and production. Both the environments have independent databases. While in development, it is mandatory to use the base URL of the development environment in --- ?
+
+The language bindings are in cURL presently with node.js and PHP bindings in development. The API documentation is set up with a display area on the right to show examples.
```

Comment:
you should do all three now

Reply:
Converted using online converters. Please verify.